### DISCUSSION TITLE
Suricata does not generate events from test-PCAPs from docs

### DISCUSSION THREAD

**Discussion by sempervictus** [Security-Onion-Solutions/securityonion#8413] [Unsupported Versions] [answered]
Using the 2nd PCAP from https://docs.securityonion.net/en/2.3/pcaps.html, i'm getting no suricata events generated, only zeek.
Running 2.3.140, with a valid oinkcode, i'm somewhat concerned that the 'forcing v6' note during rule update indicates that the rules being pulled won't work in my installation.
Does anyone have the current version running with commercial rules? How does one "roll-back" to the Open Source rules for testing?

**Comment by reyesj2** [comment]
Sounds like your SO is connected to the internet. In that case you can run so-test to get some pcap and run it through you sensor.

Is there any errors in docker logs so-suricata? Or can you see anything in the suricata eve.json file in /nsm/suricata

I have experience with commercial rules and don't have any issues with a airgapped .140 SO servers

**Comment by TotieBash** [comment]
"How does one "roll-back" to the Open Source rules for testing?"  
https://docs.securityonion.net/en/2.3/rules.html?highlight=emerging%20threat
/opt/so/saltstack/local/pillar/minions/{hostname}.sls
![image](https://user-images.githubusercontent.com/86748904/182007679-28a06e4a-7eb0-49d8-bc4a-b189c8402366.png)

As far as your suricata events.  Please show me the output of this command:
`cat /opt/so/saltstack/local/pillar/global.sls | grep hnmanager`

**Comment by sempervictus** [comment, marked as answer]
Thanks for the instructions, it was, in fact, the TALOS subscription just outright breaking suricata. Looks like we'll have to buy ETPRO for production efforts. Soon as i went to ETOPEN everything started to work
